lib/shipcloud.rb

```diff
@@ -52,7 +52,7 @@ def self.configure
 
   def self.api_headers
     API_HEADERS.merge(
-      "Affiliate-ID" => configuration.affiliate_id || DEFAULT_AFFILIATE_ID
+      "Affiliate-ID" => configuration.affiliate_id || DEFAULT_AFFILIATE_ID,
```

Comment:
I think that line/merge is redundant now, as the merge is done in [Line 31 in lib/shipcloud/request/connection.rb](https://github.com/shipcloud/shipcloud-ruby/pull/38/files#diff-8b781b79018ef926e2ddc174592a888fR31)? Or am I missing something?

Reply:
We have three possible variants of `affiliate_id`. First is `DEFAULT_AFFILIATE_ID`, second comes from the initialiser configuration, and the third belongs to an entity, which uses shipcloud, and is different from the second one, since different entities use different affiliate IDs under the same initialiser. Here we deal with the two of them, and the third one is covered in the [Line 31 in lib/shipcloud/request/connection.rb](https://github.com/shipcloud/shipcloud-ruby/pull/38/files#diff-8b781b79018ef926e2ddc174592a888fR31).

Reply:
I'm not sure about removing [Line 55 from shipcloud.rb](https://github.com/shipcloud/shipcloud-ruby/pull/38#discussion-diff-303446111L55), because if this method is called somewhere without a http request, it will return headers without `Affiliate-ID`.

Reply:
Okay, let's use `Shipcloud.affiliate_id` here for now:

```suggestion
      "Affiliate-ID" => affiliate_id,
```

This should reduce the duplication and I will have look at the default affiliate id handling later on.